### ISSUE TITLE
Lessen bid amount weight

### ISSUE THREAD

**Issue by seanyesmunt** [lbryio/lighthouse#132]
When making a search for `"LBRY is quickly gaining adoption"` (to fetch related content), which resides at `lbry://five`, the first result is `lbry://two`, which is `"Mary Jane- Her First Time Doing Porn 33 min"` and is not related at all.

This porn result comes in higher than the actual claim for this file because of it's high bid amount.

**Comment by lyoshenka** [comment]
Do we have the right approach to improving search in general? It feels like what we're doing now is finding one or a few examples of bad results, then moving the weights around a bit to get a better result. Has this been fruitful so far? Should we be doing something else? How do more established search engines work?

**Comment by seanyesmunt** [comment]
This can probably be put on hold. After some discussion it sounds like the search KPIs should be figured out first.

**Comment by tiger5226** [comment]
I agree @lyoshenka. That has been my train of thought as well. It has not been very fruitful in my opinion. We really can't keep tweaking weights in this manner. As @seanyesmunt stated, the KPIs need to be done before we tweak more weights. @tzarebczan made a final tweak to just lessen the weight much more so it is less of an issue going forward. 

The KPIs need to be broad to avoid this scenario. We need to have a measurement of the impact of changing these results. I don't have a path forward with the KPIs right now. I need to do more research.

**Comment by kauffj** [comment]
I think a very simple KPI for search is to record three things:

- The search version (and/or timestamp)
- The search term
- Was the user satisfied (yes or no)

**Comment by eggplantbren** [comment]
It's probably a good idea to use log(bid + 0.1) or some similar nonlinear function of the bid for this.
